resources/languages/fr/rules/time.clj

```diff
@@ -354,6 +354,28 @@
   #"(?i)fin de matin[ée]e"
   (assoc (interval (hour 10 false) (hour 12 false) false) :form :part-of-day :latent true)
 
+  "au déjeuner"
+  #"(?i)(pendant(le )?|au)? d[eéè]jeuner"
```

Comment:
"pendant le déjeuner" is not matched, I think there is a mixup in the regex with parenthesis can you please check? (at least, missing space in front of "le")